DuckDuckGo/UserText.swift

```diff
@@ -22,13 +22,9 @@ import Core
 
 
 public struct UserText {
-    
-    public static let webPageFailedLoad = NSLocalizedString("web.page.load.failed", comment: "DuckDuckGo cannot load this page because...")
 
     public static let appUnlock = NSLocalizedString("app.authentication.unlock", comment: "Unlock DuckDuckGo")
-    public static let homeLinkTitle = NSLocalizedString("home.link.title", comment: "DuckDuckGo Home")
     public static let searchDuckDuckGo = NSLocalizedString("search.hint.duckduckgo", comment: "Search or enter address")
-    public static let webSessionCleared = NSLocalizedString("web.session.clear", comment: "Session cleared")
```

Comment:
Some of these strings are still listed in `Localizable.strings` - could you remove these from there?